publications-2021.bib

```diff
@@ -36,5 +36,25 @@ @Article{dealii2019design
   Url                      = {https://arxiv.org/abs/1910.13247}
 }
 
+@article{samrock2021integrated,
+  title={Integrated magnetotelluric and petrological analysis of felsic magma reservoirs: Insights from Ethiopian rift volcanoes},
+  author={Samrock, Friedemann and Grayver, Alexander V and Bachmann, Olivier and Karakas, {\"O}zge and Saar, Martin O},
+  journal={Earth and Planetary Science Letters},
+  volume={559},
+  pages={116765},
+  year={2021},
+  publisher={Elsevier}
+}
+
+@article{grayver2021td,
+author = {Grayver, Alexander V. and Kuvshinov, Alexey and Werthm{\"u}ller, Dieter},
+title = {Time-domain modeling of 3-D Earth's and planetary electromagnetic induction effect in ground and satellite observations},
+journal = {Journal of Geophysical Research: Space Physics},
+volume = {129},
+pages = {e2020JA028672},
```

Comment:
Are these pages correct? I don't understand why this is not a pair of numbers.

Reply:
It is in the "published online" stage, where the pages are not known yet, but the publisher gives this thing... (I exported ciation from their website...)

Reply:
Weird. Well, if the CI doesn't complain lets go for it.